cumulus/test/runtime/Cargo.toml

```diff
@@ -77,3 +93,5 @@ std = [
 	"substrate-wasm-builder",
 ]
 increment-spec-version = []
+
+experimental = ["pallet-aura/experimental"]
```

Comment:
```suggestion
```

This feature doesn't do anything anymore. Please also remove it from the pallet Cargo.toml